tips_and_tricks/tips_and_tricks.md

```diff
@@ -11,9 +11,9 @@ prev:
 In this section we will present some CoCalc features and useful tricks that will make the management of your project easier.
 
 ## Text fields generally support markdown and LaTeX
-CoCalc not only facilitates the creation of LaTex documents, but the multiple text sections in CoCalc support and render LaTex and markdown  (specifically  [GitHub Flavored Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)).
+CoCalc not only facilitates the creation of LaTeX documents, but most input areas in CoCalc support and render LaTeX and markdown  (specifically  [GitHub Flavored Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)).
 
-For example, you can use LaTex syntax in the chat rooms:
+For example, you can use LaTeX syntax in the chat rooms:
```

Comment:
Here and above you should replace "LaTeX" by "LaTeX math formulas".  It's critical.  Otherwise, people think you can use latex features like enumerate, bibliographies, cross ref, etc.